.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
     - CODACY_PROJECT_TOKEN=$CODACY_PROJECT_TOKEN
 addons:
   apt:
-    update: true
+    # update: true
```

Comment:
Why comment this line ?

Reply:
Error while updating sources from Travis for google-chrome-stable